### PR TITLE
Add notes about keeping keys and tokens secure

### PR DESCRIPTION
**Pull request by pwoznic**
Updated several topics with a note about keeping API Keys and OAuth tokens secure.

### CODE REVIEW

File: www/docs/api-reference/auth-apis/api-keys.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ having an accidental publication of an over-privileged API key is often
 organizationally "expensive."  In general, it's recommended that you use
 [OAuth 2.0](OAuth 2.0) if/where possible for production applications.
 
+:::warning
+
+:lock: Always keep your API Keys and 0Auth tokens private. Do not share them through email, Slack, Discord, forums, or other public channels because it can lead to unauthorized access. Treat these keys with the same confidentiality as your personal credentials.
```
Comment (eskibars) [inline www/docs/api-reference/auth-apis/api-keys.md:21]: 1. Can you fix the width of this to ~80 characters?
2. It should be OAuth, not 0Auth

File: www/docs/api-reference/auth-apis/oauth.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ OAuth 2.0 has several advantages over API keys or simple usernames/passwords:
 - OAuth 2.0 is inherently more tightly scoped than API keys
 - JWT tokens are detected by many security scanning tools, allowing them to more easily be flagged in the case of accidental publication
 
+:::warning
+
+:lock: Always keep your OAuth tokens private. Do not share them through email, Slack, Discord, forums, or other public channels because it can lead to unauthorized access. Treat these tokens with the same confidentiality as your personal credentials.
```
Comment (eskibars) [inline www/docs/api-reference/auth-apis/oauth.md:27]: Please wrap fix the width of this one as well

File: www/docs/api-reference/indexing-apis/core_indexing.md [hunk 3]
```diff
@@ -19,12 +19,9 @@ The full definition of the gRPC interface is covered below.
 
 ### Service
 
-The indexing service operates in two modes: _incremental_ and _batch_. In
-incremental mode, individual documents or messages are sent to be indexed. In
-a short period of time, generally a few minutes, the new content will become
-available in the search index.
+The indexing service operates by accepting individual documents or messages to be indexed. In a short period of time, generally a few minutes, the new content will become available in the search index.
```
Comment (eskibars) [inline www/docs/api-reference/indexing-apis/core_indexing.md:22]: Please wrap this line

**Review by eskibars — CHANGES_REQUESTED** [review]
Mostly minor.  0Auth vs OAuth is the only really important one to make

File: www/docs/api-reference/auth-apis/api-keys.md (hunk 1 above)
Comment (pwoznic) [reply www/docs/api-reference/auth-apis/api-keys.md]: Done :)

File: www/docs/api-reference/auth-apis/oauth.md (hunk 2 above)
Comment (pwoznic) [reply www/docs/api-reference/auth-apis/oauth.md]: Done

File: www/docs/api-reference/indexing-apis/core_indexing.md (hunk 3 above)
Comment (pwoznic) [reply www/docs/api-reference/indexing-apis/core_indexing.md]: Done

**Comment by pwoznic** [comment]
Completed changes in latest commit

**Comment by eskibars** [comment]
Thanks, LGTM now!

**Comment by pwoznic** [comment]
It's asking me to resolve a conflict (the area where I fixed the wrapping) but it's not letting me do anything when I go to Resolve conflicts. I double-checked my editing tool and the file is not in some pending state, and git status says there is nothing to commit. I suspect it's user error but I am unable to squash and merge.

**Review by eskibars — APPROVED** [review]

**Comment by eskibars** [comment]
I just manually re-approved @pwoznic

**Comment by eskibars** [comment]
Also, I should have manually fixed the merge.  You'll want to update local branch(es).  I think what happened here was you issued a PR from your personal GH account/branch, and then you made some edits to `main` on the `vectara` account and it ended up in a conflict.  You need to make sure to make edits only to your personal account+branch and then push changes to just that branch to avoid these types of conflicts.  It should auto-update the PR here

**Pull request closed without merging**